substrate/primitives/metadata-ir/src/lib.rs

```diff
@@ -86,7 +86,7 @@ mod test {
 				call_ty: meta_type::<()>(),
 				signature_ty: meta_type::<()>(),
 				extra_ty: meta_type::<()>(),
-				signed_extensions: vec![],
+				extensions: vec![],
```

Comment:
@jsdw pinging you to approve the changes in this folder.

Reply:
This renames some stuff in the MetadataIR by the looks of it, but doesn't alter the format of metadata at all (ie `frame-metadata`) so I'm good with it!